### PR TITLE
Add wireframes

### PR DESCRIPTION
**Pull request by SimbaWANGU**
In this PR, the views were customized to match the wireframes

### CODE REVIEW

File: app/views/authors/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+  <div class="author_container">
+    <div class="image_placeholder">
+      <img src=<%= a.photo%> alt="" >
+    </div>
+    <div class="details">
+      <h1><%= a.name %></h1>
+      <p>Number of posts: <%= a.posts_counter %></p>
+    </div>
+  </div>
   <% end %>
```
Comment (mikethreels) [inline app/views/authors/index.html.erb:12]: - [ ] Clicking on each user should take you to the show page

File: app/views/authors/show.html.erb [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+  </div>
+  <% @author.posts.each do |a| %>
+  <div class="posts">
+    <h4>Post #<%= (@author.posts.find_index(a) + 1)  %></h4>
+    <p><%= a.text %></p>
+    <div class="stats">
+      <p>Comments: <%= a.comments_counter %>,</p>
+      <p>Likes: <%= a.likes_counter %></p>
+    </div>
+  </div>
```
Comment (mikethreels) [inline app/views/authors/show.html.erb:22]: - [ ] you should use #recent_post method to render the user's 3 most recent post

File: app/views/authors/show.html.erb [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+  <% end %>
+  <button class="button">See All Posts</button>
```
Comment (mikethreels) [inline app/views/authors/show.html.erb:24]: - [ ] the "see all posts" button is not redirecting to the post index page.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Simba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
Hi @SimbaWANGU 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- Clicking on each user takes me to their show page
- 'See all Post' link opens the user's post index
- `#recent_post` method is used to render the user's 3 most recent post
Great job 👍

### Optional suggestions
- Your readme file does not present the current project `Blog app`. Please update your readme with the proper one.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**